bundles/org.openhab.binding.fronius/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -127,4 +165,116 @@
 		<description>Battery Power ( + charge, - discharge )</description>
 		<state pattern="%.2f W" readOnly="true"></state>
 	</channel-type>
+
+
+	<channel-type id="meter_model" advanced="true">
+		<item-type>String</item-type>
+		<label>Model</label>
+		<description></description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="meter_serial" advanced="true">
+		<item-type>String</item-type>
+		<label>Serial Number</label>
+		<description>Meter's Serial Number</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="meter_enable" advanced="true">
+		<item-type>Number</item-type>
+		<label>Enabled</label>
+		<description>Enabled</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="meter_location" advanced="true">
+		<item-type>Number</item-type>
+		<label>Location</label>
+		<description>Meter location code</description>
+		<state readOnly="true"></state>
```

Comment:
You could add the state options mentioned in the readme here.

Reply:
I'm not sure it's appropriate.